server_admin/topics/users/proc-allow-user-to-delete-account.adoc

```diff
@@ -0,0 +1,52 @@
+[id="proc-allow-user-to-delete-account_{context}"]
+= Enabling Account Deletion by Users
+
+End users and applications can delete their accounts in the Account Console if you enable this capability in the Admin Console. Once you enable this capability, you can give that capability to specific users.
+
+== Enabling the Delete Account Capability
+
+You enable this capability on the *Required Actions* tab.
+
+.Procedure 
+. Click *Authentication* in the menu.
+. Click the *Required Actions* tab.
+. Select *Enabled* on the *Delete Account* row.
++
+.Delete Account on Required Actions tab
+image:images/enable-delete-account-action.png[]
```

Comment:
@andymunro I just wonder if we should include the metadata for images into the documentation, considering people with visual impairments. For example:

```
image:images/enable-delete-account-action.png[Delete account on required actions tab]
```

@hmlnarik does it make sense?

Reply:
@hmlnarik @abstractj Yes, it makes sense.  I have done that elsewhere, so thanks for catching that omission.

I created a checklist of standards based on the IBM style guide and we do want to add that metadata. I added that checklist to the JIRA, but here it is now: https://docs.google.com/document/d/1Ei2YTokkh7VfO4AwospoSRzHXjFQ_GYCKkfELSpbYYY/edit#heading=h.99n2xdpeusco

Note that some standards are different from current usage. For example, IBM guidelines are to use asterisks, not back ticks for UI elements. We need to gradually change our style